test-fixtures/transform/filter/filterByFileType/01/input.xml

```diff
@@ -7,7 +7,7 @@
   <ListRecords>
     <record>
       <header>
-        <identifier>oai:foobar.example.com:10024/456</identifier>
+        <identifier>oai:foobar.fileType.com:10024/456</identifier>
```

Comment:
Lets use example.com TLD whenever we can (e.g., fileType.example). But having integration tests like this is ok until refactor in which we will move towards utilizing more unit tests.